yarn-project/bb-prover/src/instrumentation.ts

```diff
@@ -92,7 +92,6 @@ export class ProverInstrumentation {
     const s = typeof timerOrMS === 'number' ? timerOrMS / 1000 : timerOrMS.s();
     this[metric].record(s, {
       [Attributes.PROTOCOL_CIRCUIT_NAME]: circuitName,
-      [Attributes.PROTOCOL_CIRCUIT_TYPE]: 'server',
```

Comment:
I removed these attributes because they were always set to 'server' since we're not capturing metrics from the PXE 🤷